lib/mapper.js

```diff
@@ -19,7 +19,7 @@ var time = require('unix-time');
 exports.track = function(msg, settings){
   var eventName = msg.event();
   var appEvents = settings.appEvents;
-  if (appEvents && appEvents[eventName]) {
+  if (appEvents && appEvents[eventName] && (appEvents[eventName]!=="null") ) {
```

Comment:
So this isn't an urgent issue in that grow can simply map the event correctly in the Ui and the integration will work?